web/react/utils/async_client.jsx

```diff
@@ -104,7 +104,7 @@ module.exports.updateLastViewedAt = function() {
 module.exports.getMoreChannels = function(force) {
     if (isCallInProgress("getMoreChannels")) return;
 
-    if (ChannelStore.getMoreAll().length == 0 || force) {
+    if (!ChannelStore.getMoreAll() || force) {
```

Comment:
So with this, there is no condition where getMoreAll will return a false-y value. Why the change away from length?